test_affect_layer.py

```
@@ -0,0 +1,104 @@
+    # Simulate foundation states
+    print("\n=== Test 4: Calculate Composite ===")
+    foundation_states = {
+        FoundationAffect.SEEKING: 0.8,
+        FoundationAffect.FEAR: 0.3,
+        FoundationAffect.CARE: 0.7
+    }
+    
+    intensity, threshold = engine.calculate_composite(
+        "DARING", foundation_states, domain="deep_conversations"
+    )
+    print(f"  SEEKING: 0.8, FEAR: 0.3")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)